Makefile.include

```diff
@@ -1031,7 +1031,7 @@ ifneq (, $(filter all flash, $(if $(MAKECMDGOALS), $(MAKECMDGOALS), all)))
 endif
 
 help:
-	@$(MAKE) -qp | sed -ne 's/\(^[a-z][a-z_-]*\):.*/\1/p' | sort | uniq
+	@$(MAKE) -qp | sed -ne 's/\(^[a-z][a-zA-Z._-]*\):.*/\1/p' | grep -Fv -e '.module' -e '.xml' | sort | uniq
```

Comment:
Please add a comment as to what this incantation is good for

Reply:
Added and force-pushed.

Reply:
you could move the grep as additional `-d` to sed. not sure that makes it more readable or measurably faster.
```suggestion
	@$(MAKE) -qp | sed -e '/\.module$/d' -e '/\.xml$/d' -ne 's/\(^[a-z][a-zA-Z._-]*\):.*/\1/p' | sort | uniq
```

My `sort` has an `-u` flag that further allows removing uniq.